data/versions.yaml

```diff
@@ -3,10 +3,12 @@
 # They can be referred to with the custom Hugo shortcode, e.g. {{% version "cucumberjvm" %}}
 #
 
-cucumberjvm: "4.7.4"
-cucumberarchetype: "4.8.0.0"
+cucumberjvm: "4.8.0"
```

Comment:
Afaik the archetype does need the last `.0` (in case we update the archetype for the version)